sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -147,31 +146,53 @@ public class AppConfigurationTestEnvironment : TestEnvironment
 }
 ```
 
+### Running live tests serially
```

Comment:
Thanks for adding this - I've been wanting to know how to do this for the ACR library!

Reply:
+1 - I'm going to add this for Service Bus which will hopefully address some of the flakiness in the Functions extensions tests.